VideoLocker/res/drawable/white_circle.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+    <solid android:color="@color/white" />
```

Comment:
We should deprecate this color in favor of `edx_grayscale_neutral_white_t` as its the one in the edX color palette for white color.